apps/src/templates/projects/projectsRedux.js

```diff
@@ -104,6 +105,18 @@ function personalProjectsList(state = initialPersonalProjectsList, action) {
         ...state,
         projects: updatedProjects
       };
+    case UNPUBLISH_SUCCESS:
+      var unpublishedChannel = action.channel;
```

Comment:
i would use `const` instead of `var` for the variable declarations on this line, 111, and 113